cypress/e2e/app/simple-e2etest.cy.js

```diff
@@ -70,44 +72,50 @@ describe("End to end test using a simple UI path through the app", () => {
                 // B. Categorize "participant_id" as "Subject ID"
                 cy.categorizeColumn("Subject ID", p_dataset["category_columns"]["Subject ID"][0]);
 
-                // C. Assert nav and next button are enabled
-                cy.assertNextPageAccess("annotation", true);
+                // C. Assert nav and next button are not yet enabled
+                cy.assertNextPageAccess("annotation", false);
 
                 // D. Categorize "age" as "Age"
                 cy.categorizeColumn("Age", p_dataset["category_columns"]["Age"][0]);
 
+                cy.assertNextPageAccess("annotation", true);
```

Comment:
same as above: I would now expect a hint on why we want the assertion to be true.